src/content/developers/tutorials/how-to-view-nft-in-metamask.md

```diff
@@ -0,0 +1,28 @@
+
+# How to View Your NFT in Your Wallet (Part 3/3 of NFT Tutorial Series)
+
+This tutorial idescribes how to view an existing an NFT on your virtual Ethereum wallet (Metamask in this case)! If you’d like to learn how to mint your own NFT on Ethereum, you should check out Part 1 on How to Write & Deploy an NFT smart contract!
```

Comment:
Link this to part 1

Reply:
This may sound kind of silly, but how do I link it to part 1 before publishing??